packages/react-components/react-components/src/AccessibilityScenarios/Card.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+
+import { Card, CardHeader, CardFooter } from '@fluentui/react-components/unstable';
+import { Button } from '@fluentui/react-components';
+
+import { Scenario } from './utils';
+
+type FocusModeType = 'off' | 'no-tab' | 'tab-exit' | 'tab-only';
+
+interface CardTemplateProps {
+  cardId: string;
+  name: string;
+  lastSeen: string;
+  focusMode: FocusModeType;
+}
+const CardTemplate: React.FunctionComponent<CardTemplateProps> = ({ cardId, name, lastSeen, focusMode }) => {
+  const messageDescId = `${cardId}-message-desc`;
+  return (
+    <Card focusMode={focusMode} aria-label={`${name} card`}>
+      <CardHeader
+        image={{ as: 'img', src: '#', alt: 'Face of a person' }}
+        header={{ as: 'h3', children: name }}
+        description={`Last seen ${lastSeen} ago.`}
+      />
+      <CardFooter>
+        <Button aria-describedby={messageDescId}>Private message</Button>
+        <span id={messageDescId} style={{ display: 'none' }}>
+          Send a private message to {name}.
+        </span>
+        <Button aria-haspopup="menu">More options</Button>
+      </CardFooter>
+    </Card>
+  );
+};
+
+interface CardGroupProps {
+  groupId: string;
+  focusMode: FocusModeType;
+}
+const CardGroup: React.FunctionComponent<CardGroupProps> = ({ groupId, focusMode }) => (
+  <>
+    <CardTemplate cardId={`card1-${groupId}`} name="Bruce wayne" lastSeen="4 hours" focusMode={focusMode} />
+    <CardTemplate cardId={`card2-${groupId}`} name="Clark Kent" lastSeen="30 minutes" focusMode={focusMode} />
+    <CardTemplate cardId={`card3-${groupId}`} name="Peter Parker" lastSeen="2 days" focusMode={focusMode} />
```

Comment:
I have added a selectable variant to the same Card component scenario. Are there some exammples of image-only, no-image, action button, and interactive cards based on which I can design other variations? Or perhaps, are those variants only the same cards but the content differs? In that case I thought that there is no reason to add more examples because the cards behave the same.